json-schemas/definitions.json

```diff
@@ -14,6 +14,10 @@
     "did": {
       "type": "string",
       "pattern": "^did:([a-z0-9]+):((?:(?:[a-zA-Z0-9._-]|(?:%[0-9a-fA-F]{2}))*:)*((?:[a-zA-Z0-9._-]|(?:%[0-9a-fA-F]{2}))+))((;[a-zA-Z0-9_.:%-]+=[a-zA-Z0-9_.:%-]*)*)(\/[^#?]*)?([?][^#]*)?(#.*)?$"
+    },
+    "date-time": {
+      "type": "string",
+      "pattern": "^[0-9]{4}-[0-1]\\d-[0-3]\\dT([0-2]\\d:[0-5]\\d:[0-5]\\d|23:59:60)\\.[0-9]{6}Z$"
```

Comment:
why not `\\d{4}` and `\\d{6}`?

also i we probably should put `?:` at the start of every capture group just so that we aren't actually capturing anything.  `(?:[0-2]\\d:[0-5]\\d:[0-5]\\d|23:59:60)`

Reply:
Good suggestion on `?:`! On it!